transform/mattermost-analytics/models/marts/release/dim_releases.sql

```diff
@@ -1,4 +1,17 @@
+with release_versions as (
+    select
+        short_version,
+        lag(short_version, 1) over(order by release_number asc) as previous_release_version,
+        lag(short_version, -1) over(order by release_number asc) as next_release_version
+    from
+        {{ ref('stg_mattermost__version_release_dates') }}
+    where
```

Comment:
I assume `version_patch = 0` filter was superfluous in the prev. iteration?

Reply:
It's pretty much the same with releases having an RC1 date. 

Perhaps we can also add a test @ the source checking that whenever rc1 date is not null, patch_version is equal to 0.